### ISSUE TITLE
Input/Output dimensions

### ISSUE THREAD

**Issue by iiSeymour** [lucidrains/feedback-transformer-pytorch#1]
Hey @lucidrains 

Can I check the dimensions of the input and output, is it `(seq_len, dim)` -> `(? ,dim, tokens)`?

```python
model = FeedbackTransformer(
    num_tokens = 20000,           # number of tokens
    dim = 512,                    # dimension
    depth = 6,                    # depth
    seq_len = 2,                  # the sequence length of each segment or window
    mem_len = 256,                # length of the memory buffer
    dim_head = 64,                # dimension of each head
    heads = 8,                    # number of heads
    attn_dropout = 0.1,           # attention dropout
    ff_dropout = 0.1              # feedforward dropout
).cuda()

x = torch.randint(0, 256, (2, 512)).cuda()
model(x)  # (1, 512, 20000)
```

**Comment by lucidrains** [comment]
@iiSeymour Hi Chris! The input is (batch x sequence length) and the output is (batch x sequence length x logit dimension [number of tokens])

I've cleared it up in the readme :pray:
